README.md

```diff
@@ -1,7 +1,91 @@
-# Fabric
+# Fabric API
 
-Essential hooks and patches for modding with Fabric.
+Essential hooks for modding with Fabric.
 
-## Getting Started
+Fabric API is the library for essential hooks and interoperability mechanisms for Fabric mods. Examples include:
 
-To setup a Fabric development environment, check out the [example fabric mod](https://github.com/FabricMC/fabric-example-mod) and follow the instructions there.
+- Exposing functionality that is useful but difficult to access for many mods such as particles, biomes and dimensions
+- Adding events, hooks and APIs to improve interopability between mods.
+- Essential features such as registry synchronization and adding information to crash reports.
+- An advanced rendering API designed for compatibility with optimization mods and graphics overhaul mods.
+
+Also check out [Fabric Loader](https://github.com/FabricMC/fabric-loader), the (mostly) version-independent mod loader that powers Fabric. Fabric API is a mod like any other Fabric mod which requires Fabric Loader to be installed.
+
+For support and discussion for both developers and users, visit [the Fabric Discord server](https://discord.gg/v6v4pMv).
+
+## Using Fabric API
+
+To use Fabric API, download it from [CurseForge](https://www.curseforge.com/minecraft/mc-mods/fabric-api) or [GitHub Releases](https://github.com/FabricMC/fabric/releases).
```

Comment:
I think this needs to describe the kind of use, i.e. playing with fabric mods vs using fabric api for developing mods.